ytmusicapi/helpers.py

```diff
@@ -96,7 +96,7 @@ def get_datestamp():
 
 
 def to_int(string):
-    number_string = re.split('[\x20\xa0]', string)[0]
+    number_string = re.sub('[^\\d]', '', string)
```

Comment:
Seems reasonable, but is there a reason why you included this with the Korean language PR?

Reply:
because, the word order of korean is different from others